sysdig/internal/client/v2/model.go

```diff
@@ -411,6 +413,35 @@ type CloudAccountSecure struct {
 	WorkLoadIdentityAccountID    string `json:"workloadIdentityAccountId,omitempty"`
 	WorkLoadIdentityAccountAlias string `json:"workLoadIdentityAccountAlias,omitempty"`
 }
+
+type CloudAccountFeature struct {
+	// TODO: define fields based on how features are passed in TF resource,
+	// and in sync with modified cloudAccount API request body
+}
+
+type CloudAccountFeatures struct {
+	SecureThreatDetection     *CloudAccountFeature
+	SecureConfigPosture       *CloudAccountFeature
+	SecureIdentityEntitlement *CloudAccountFeature
+	MonitorCloudMetrics       *CloudAccountFeature
+	SecureAgentlessScanning   *CloudAccountFeature
+}
+
+type CloudAccountSecureV2 struct {
```

Comment:
Analyzing if we can import and use the Cloud Account object directly from `draios/protorepo/cloudauth`?
And then the `*FromResourceData` and `*ToResourceData` in resource*.go can do the from/to conversion of the fields as required.

Reply:
Done.